GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAndroidResource.java

```diff
@@ -308,7 +327,7 @@ public synchronized final InputStream getStream() throws IOException{
      */
     public synchronized final void closeStream() {
         try {
-            if (streamState == StreamStates.OPEN) {
+            if ((stream != null) && (streamState == StreamStates.OPEN) {
```

Comment:
What is this case where the state is OPEN but there is no stream? Is it a valid state to be in?